journal.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+import os
+import datetime
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+
+
+# DATABASE_URL = os.environ.get(
+#     'DATABASE_URL',
+#     'postgresql://jameshemmaplardh:123@localhost:5432/learning-journal'
+# )
+
+
+
+Base = declarative_base()
+
+@view_config(route_name='home', renderer='string')
+def home(request):
+    return "Hello World"
+
+
+def main():
+    """Create a configured wsgi app"""
+    settings = {}
+    debug = os.environ.get('DEBUG', True)
+    settings['reload_all'] = debug
+    settings['debug_all'] = debug
+    # configuration setup
+    config = Configurator(
+        settings=settings
+    )
+    config.add_route('home', '/')
+    config.scan()
+    app = config.make_wsgi_app()
+    return app
+
+DATABASE_URL = os.environ.get(
```

Comment:
module-level constants like this should generally be bound at the top of the file, not down here.  This should actually be moved back up to where the commented-out version is, and this version should be removed.